src/apps/queues/rabbit.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+
+from pyrabbit.api import Client
+
+from django.conf import settings
+from pyrabbit.http import HTTPError, NetworkError
+
+
+def _get_rabbit_connection():
+    """Helper giving us a rabbit connection from settings.BROKER_URL"""
+    host_with_port = "{}:{}/".format(settings.RABBITMQ_HOST, settings.RABBITMQ_MANAGEMENT_PORT)
+    # TODO: Scheme was deprecated? Verify this
```

Comment:
this TODO?

Reply:
Tested this and I'm not 100% but I think I can pull it. Signature of the class doesn't seem to take it, and it works with/without...